amgut/test/test_dbpatch_source.py

```diff
@@ -0,0 +1,104 @@
+from unittest import TestCase, main
```

Comment:
You are right in all regards: 1) it is a step backward AND 2) we do have these improperly operating tests. Therefore, I suggest to keep this test until we are sure that both amgut and labadmin cleans properly after every test.